Explorer/Assets/DCL/Notifications/NotificationsMenu/NotificationsMenuController.cs

```diff
@@ -95,6 +115,28 @@ private async UniTaskVoid InitialNotificationRequestAsync(CancellationToken ct)
                 notifications.Add(requestNotification);
 
             view.LoopList.SetListItemCount(notifications.Count, false);
+
+            foreach (var notification in requestNotifications)
+                if (notification.Read == false)
+                    unreadNotifications++;
+
+            UpdateUnreadNotificationRender();
+        }
+
+        private void UpdateUnreadNotificationRender()
+        {
+            view.unreadNotificationCounterText.text = unreadNotifications.ToString();
```

Comment:
To String allocates a new string, you can improve it with the SetText provided by text mesh pro IIRC
```
 view.unreadNotificationCounterText.SetText("{0}", unreadNotifications);
```